### PR TITLE
Add data

### PR DESCRIPTION
**Pull request by namphong124563**
Add my name and image through Github Desktop

- [x] Included a Preview link and screenshot showing after and before the changes.
- [x] Included a description of the change below.
- [x] Squashed the commits.

# Changes done in this Pull Request

- [Preview Link](https://namphong124563.github.io/gci18.fossasia.org/)

- Fixes #<Issue Number>

## Description / Changes
- Add my name and student image.

### After:

- - - - - - - - - - - -

### CODE REVIEW

File: _data/students.yml [hunk 1]
```diff
@@ -165,3 +165,6 @@
   github: Yash-Garg
   image: yashg.png
 
+- name: 三銃士
```
Comment (abishekvashok) [inline _data/students.yml]: Can you make this up in English pls?

Comment (namphong124563) [reply _data/students.yml]: I though UTF-8 character can be shown on the website?

Comment (abishekvashok) [reply _data/students.yml]: It can, but if you could add it to English it would make it world readable.

Comment (namphong124563) [reply _data/students.yml]: Finished.

Comment (namphong124563) [reply _data/students.yml]: Could you merge the commits for me or show me how to merge the commits?

**Pull request merged**